src/components/Home/Map/Search.js

```diff
@@ -87,17 +97,27 @@ export default function Search({ zoomOnCluster }) {
       >
         {geoRes.features && geoInput.length >= MIN_QUERY_LENGTH
           ? geoRes.features.map((f, i) => (
-            <Option key={nanoid()} value={`${i}`}>
-              {f.place_name}
-            </Option>
-          ))
+              <Option key={nanoid()} value={`${i}`}>
+                {f.place_name}
+              </Option>
+            ))
           : ''}
       </AutoComplete>
       <RangePicker
         style={{ width: '100%' }}
         disabledDate={disableFutureDates}
         onCalendarChange={handleCalendarChange}
       />
+      <div>
+        <label>
+          Quick Search
+          <select onChange={onChange} value={quickSelect} name="quicksearch">
+            <option value="">- Select an option -</option>
```

Comment:
Maybe have the default option be something like "No time filter" or "show all data" (neither of those sounds too great to me but you get the point) so the user knows they don't have to select a time frame filter.